### ISSUE TITLE
Bad case on external image

### ISSUE THREAD

**Issue by karthikg-09** [AuroBit/ComfyUI-OOTDiffusion#21]
please update [onnx support for human parsing] which will give good results compared to the current.
Thank you for this awesome node

**Comment by iyume** [comment]
onnx is just a replacement for pth which shouldn't have a better result compared to the current. But it seems to solve some compiler problems like `inplace_abn`. See https://github.com/AuroBit/ComfyUI-OOTDiffusion/issues/8

I will add and test it later.

**Comment by karthikg-09** [comment]
![image](https://github.com/AuroBit/ComfyUI-OOTDiffusion/assets/153519785/75b098a7-634b-4624-ad35-22d1abe24374)
![green shirt from repo](https://github.com/AuroBit/ComfyUI-OOTDiffusion/assets/153519785/66af779b-c312-4c42-97f8-fb7a038c6e16)
can you see the difference between comfy and gradio implementation?
after the new update I think both full and half body results are improved a lot

**Comment by iyume** [comment]
onnx support has been added.
For your use case, I believe it's a corner case. You can try generating multiple times at https://ootd.ibot.cn/ and observe the bad case.
If it was still a problem, please provide the link to your test images.

![test](https://github.com/AuroBit/ComfyUI-OOTDiffusion/assets/48091591/16cd260a-280f-4a6c-b0e4-c6415aff8e71)

**Comment by karthikg-09** [comment]
after multiple runs and comparison, i feel demo works much better (in first generation only).. for some complex clothes
![man-white](https://github.com/AuroBit/ComfyUI-OOTDiffusion/assets/153519785/f88ec3a8-5cef-4753-ae4d-3d64cd52e3c5)
![mens-shirts](https://github.com/AuroBit/ComfyUI-OOTDiffusion/assets/153519785/3c804d46-ab2a-43cb-95fd-644932a529b2)
![shirt](https://github.com/AuroBit/ComfyUI-OOTDiffusion/assets/153519785/0b252c5a-3732-426f-945c-c6c50923eddb)
where can I contact you?

**Comment by iyume** [comment]
Yes, I have noticed that official demo works better. I will try to figure it out (by deploy the official gradio code).

**Comment by karthikg-09** [comment]
@iyume Thank you.. I will be waiting for your update

**Comment by PrinceDhankhar** [comment]
Any Update on This?

**Comment by makisekurisu-jp** [comment]
Waiting for update
